README.adoc

```diff
@@ -61,21 +61,22 @@ yarn prepare:admin
 . Start the server
```

Comment:
I think we will eventually do single script that will run docker-compose, build all clients and start servers. This way it will be easy to run it by judges :D